.schema/auditlog.schema.json

```diff
@@ -0,0 +1,94 @@
+{
+  "title": "Oathkeeper Audit Log Configuration",
+  "type": "array",
+  "items": {
+    "$ref": "#/definitions/EventHandler"
+  },
+  "default": [],
+  "uniqueItems": true,
+  "definitions": {
+    "EventHandler": {
+      "description": "Event is a handler for single event",
```

Comment:
should the phrase "Event is a handler for..." be "EventHandler is a handler for ...."?